proxy/pkg/metrics/proxy_metrics.go

```diff
@@ -5,6 +5,9 @@ const (
 	typeReadsTarget = "reads_target"
 	typeWrites      = "writes"
 	typeCount       = "count"
+	typeIdsOrigin   = "ids_origin"
+	typeIdsTarget   = "ids_target"
+	typeIdsAsync    = "ids_async"
```

Comment:
I would remove the `ids_` part, `connector=origin` makes more sense than `connector=ids_origin`

Reply:
Done